compose-files/docker-compose.yml

```
@@ -56,7 +56,7 @@ services:
     command: --bind_ip_all --replSet rs0
 
   mongosetup:
-    image: mongo:5.0.26
+    image: mongo:6.0.15
```

Last version after docker hub is 6.0.16 
```suggestion
    image: mongo:6.0.16
```